diff --git a/server/src/ghostlab/GameServer.java b/server/src/ghostlab/GameServer.java
index 5ecaaa1..2c45f21 100644
--- a/server/src/ghostlab/GameServer.java
+++ b/server/src/ghostlab/GameServer.java
@@ -271,6 +271,12 @@ public class GameServer {
     }
     multicast.ENDGA(id, maxScore);
 
+    try {
+      Thread.sleep(2000);
+    } catch (InterruptedException e1) {
+      Logger.log("[-] Thread sleeping failed");
+    }
+
     for (Player p : lobby) {
       try {
         p.TCPSocket.close();
